CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Entries in this file should be limited to:
 Please avoid adding duplicate information across this changelog and JIRA/doc input pages.
 
 ## [NEXT RELEASE]
+- ROX-11348: The email notifier now allows for unauthenticated smtp. By default, unauthenticated smtp is turned off.
```

Comment:
"unauthenticate" and "turned off" is a little confusing, a sort of double-negative.

```suggestion
- ROX-11348: The email notifier now allows for unauthenticated SMTP. By default, authentication is still required for an email notifier, but the user can now turn it off.
```